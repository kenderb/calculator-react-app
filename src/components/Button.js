import React from 'react';
import PropTypes from 'prop-types';

const Button = ({ value }) => (<button type="button">{value}</button>);
Button.propTypes = {
  value: PropTypes.string.isRequired,
};
export default Button;
