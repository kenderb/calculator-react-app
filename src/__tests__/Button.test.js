import { create } from 'react-test-renderer';
import Button from '../components/Button';

test('renders correctly', () => {
  const tree = create(<Button onClickBtn={() => 'hello'} buttonName="name" />).toJSON();
  expect(tree).toMatchSnapshot();
});
