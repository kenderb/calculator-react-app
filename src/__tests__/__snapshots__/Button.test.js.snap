// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="sc-dlfnbm fLGXYK"
  color="false"
  onClick={[Function]}
  type="button"
>
  name
</button>
`;
